utility/src/test/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsAssessorImplTest.kt

```diff
@@ -32,152 +37,221 @@ import org.oppia.android.util.logging.GlobalLogLevel
+      .write(ByteArray(ONE_KILOBYTE))
+    assertThat(performanceMetricsAssessorImpl.getDeviceStorageTier())
+      .isEqualTo(OppiaMetricLog.StorageTier.LOW_STORAGE)
   }
 
   @Test
-  fun testPerformanceMetricsUtils_getTotalStorageUsageTier_returnsCorrectStorageUsageTier() {
-    context.openFileOutput(TEST_FILE_NAME, Context.MODE_PRIVATE).use {
-      it.write(TEST_FILE_CONTENT.toByteArray())
-    }
-    val cacheFile = File.createTempFile(TEST_APP_PATH, null, context.cacheDir)
-    val expectedStorageValue = TEST_FILE_CONTENT.length + cacheFile.length()
+  fun testPerformanceMetricsUtils_writeBytesEqualToLowTierHigherBound_retsLowStorageUsageTier() {
+    context.openFileOutput(TEST_APP_PATH, Context.MODE_PRIVATE)
+      .write(ByteArray(ONE_MEGABYTE))
+    assertThat(performanceMetricsAssessorImpl.getDeviceStorageTier())
+      .isEqualTo(OppiaMetricLog.StorageTier.LOW_STORAGE)
+  }
 
-    val expectedStorageTierValue = when (expectedStorageValue / (1024 * 1024 * 1024)) {
-      in 0L until 32L -> OppiaMetricLog.StorageTier.LOW_STORAGE
-      in 32L..64L -> OppiaMetricLog.StorageTier.MEDIUM_STORAGE
-      else -> OppiaMetricLog.StorageTier.HIGH_STORAGE
-    }
+  @Test
+  fun testPerformanceMetricsUtils_writeBytesEqualToMedTierLowerBound_retsMediumStorageUsageTier() {
+    context.openFileOutput(TEST_APP_PATH, Context.MODE_PRIVATE)
+      .write(ByteArray(TWO_MEGABYTES.toInt()))
+    assertThat(performanceMetricsAssessorImpl.getDeviceStorageTier())
+      .isEqualTo(OppiaMetricLog.StorageTier.MEDIUM_STORAGE)
+  }
 
-    assertThat(performanceMetricsUtilsImpl.getDeviceStorageTier())
-      .isEqualTo(expectedStorageTierValue)
+  @Test
+  fun testPerformanceMetricsUtils_writeBytesEqualToMedTierHigherBound_retsMediumStorageUsageTier() {
+    context.openFileOutput(TEST_APP_PATH, Context.MODE_PRIVATE)
+      .write(ByteArray(THREE_MEGABYTES.toInt()))
+    assertThat(performanceMetricsAssessorImpl.getDeviceStorageTier())
+      .isEqualTo(OppiaMetricLog.StorageTier.MEDIUM_STORAGE)
+  }
+
+  @Test
+  fun testPerformanceMetricsUtils_writeBytesGreaterThanHighTierLowerBound_retsHighStorageTier() {
+    context.openFileOutput(TEST_APP_PATH, Context.MODE_PRIVATE)
+      .write(ByteArray(TWO_MEGABYTES.toInt() * 2))
+    assertThat(performanceMetricsAssessorImpl.getDeviceStorageTier())
+      .isEqualTo(OppiaMetricLog.StorageTier.HIGH_STORAGE)
   }
 
   @Test
   fun testPerformanceMetricsUtils_getBytesSent_returnsCorrectAmountOfNetworkBytesSent() {
-    val expectedNetworkBytesSent = TrafficStats.getUidTxBytes(
-      ApplicationProvider.getApplicationContext<Application>().applicationInfo.uid
-    )
+    val shadow = Shadow.extract(TrafficStats()) as OppiaShadowTrafficStats
+    shadow.setUidTxBytes(20L)
 
-    assertThat(performanceMetricsUtilsImpl.getTotalSentBytes())
-      .isEqualTo(expectedNetworkBytesSent)
+    assertThat(performanceMetricsAssessorImpl.getTotalSentBytes())
+      .isEqualTo(20L)
```

Comment:
Could fit on one line. Ditto for the others.

Reply:
done.